index.html

```diff
@@ -14,12 +15,28 @@
   <![endif]-->
 </head>
 
-<body>
-
-  
-     
+<body>  
   <div class="container home">
-   
+      <section class = "navsection">
+        <div class = "maintitle">
+          <h1>Fun Bus</h1>
+          </div>
+          <nav>
+            <a href="#home">Home</a>
+            <a href="#about">About Us</a>
+            <a href="#blog">Blog</a>
+            <a href="#contact">Contact</a>
+          </nav>
+          </section>
+          <div class = "img">
+            <img src = "img/fun-bus.jpg" alt= "bus">
```

Comment:
Same issue here. Since there's only 1 thing inside the div, you don't need the div here.